### DISCUSSION TITLE
Help on activating other query runners on self hosted

### DISCUSSION THREAD

**Discussion by JamesSSchuler** [getredash/redash#6633] [General]
I have an AWS image launched , v10, but I cannot find any clear way to add the other supported database types( MySql, etc...)  that are not activated by default.
Can anyone point me in the right direction?

**Comment by masayuki038** [comment]
Hi @JamesSSchuler

Can you click "Settings" on bottom left and "*New Data Soource"?
![image](https://github.com/getredash/redash/assets/36331/b83f861b-bb99-4224-a5e3-a26394418f04)

**Comment by JamesSSchuler** [comment]
@masayuki038   Yes, but there are quite a few that are listed as supported that dont appear ( like MySQL ), and I cant find a clear cut way to add them
This list:
https://redash.io/help/data-sources/querying/supported-data-sources

Unless I am missing some glaring documentation, I see to add to the .env file, but there is no list of the actual values to use per service
or if that is even the right way.

**Comment by masayuki038** [comment]
@JamesSSchuler Umm...

Unfortunately, I don't have AWS account. So, I can't use the image...
Instead of it, I pulled [redash/redash/10.0.0.b50363](https://hub.docker.com/layers/redash/redash/10.0.0.b50363/images/sha256-9392753c03767453f3a803eea0f6f57b52163dac8814dd13a1724dc4037132cf?context=explore) and run it. It shows MySQL data source.

![image](https://github.com/getredash/redash/assets/36331/72eb42a6-a8fa-4d2a-9075-9b128b4c07f3)
![image](https://github.com/getredash/redash/assets/36331/7fb53d7a-5bed-4c23-b7fc-dec7e7f6d372)

So, I think MySQL is supported with v10 image without any special operations...

Are there any errors in logs?

**Comment by masayuki038** [comment]
I am going to close this since registered as https://github.com/getredash/redash/issues/6634
